spk/sickchill/Makefile

```diff
@@ -0,0 +1,38 @@
+SPK_NAME = sickchill
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 0
+SPK_ICON = src/sickchill.png
+
+SPK_DEPENDS = "python3>=3.6:git"
+
+MAINTAINER = Nyaran
+DESCRIPTION = Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
+DESCRIPTION_SPN = Gestor automático para bibliotecas de series. Busca nuevos episodios de tus series favoritas, y cuando son publicados hace su magia.
+STARTABLE = yes
+DISPLAY_NAME = SickChill
+CHANGELOG = "Initial SickChill version"
+
+HOMEPAGE   = https://sickchill.github.io/
+LICENSE    = GPL
+
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+SERVICE_PORT = 8081
```

Comment:
Port 8083 so

Reply:
8083 is the default port used by the package **SickBeard-Custom**, **SickChill** uses the port 8081